steampy/login.py

```diff
@@ -51,10 +51,16 @@ def _send_login_request(self) -> Response:
     def set_sessionid_cookies(self):
         community_domain = SteamUrl.COMMUNITY_URL[8:]
         store_domain = SteamUrl.STORE_URL[8:]
-        for name in ('steamLoginSecure', 'sessionid', 'steamRefresh_steam', 'steamCountry'):
+        community_cookie_dic = self.session.cookies.get_dict(domain = community_domain)
+        store_cookie_dic = self.session.cookies.get_dict(domain = store_domain)
+        for name in ['steamLoginSecure', 'sessionid', 'steamRefresh_steam', 'steamCountry']:
             cookie = self.session.cookies.get_dict()[name]
-            community_cookie = create_cookie(name, cookie, community_domain)
-            store_cookie = create_cookie(name, cookie, store_domain)
+            if name == 'steamLoginSecure':
+                community_cookie = self._create_cookie(name, community_cookie_dic['steamLoginSecure'], community_domain)
```

Comment:
Did I miss self._create_cookie(...) method somewhere? Or it is imported from utils?

Reply:
you are not mistaken, it would be correct to use it this way

```
for name in ['steamLoginSecure', 'sessionid', 'steamRefresh_steam', 'steamCountry']:
      cookie = self.session.cookies.get_dict()[name]

      if name == 'steamLoginSecure':
          community_cookie = create_cookie(name, community_cookie_dic['steamLoginSecure'], community_domain)
          store_cookie = create_cookie(name, store_cookie_dic['steamLoginSecure'], store_domain)
      else:
          community_cookie = create_cookie(name, cookie, community_domain)
          store_cookie = create_cookie(name, cookie, store_domain)

      self.session.cookies.set(**community_cookie)
      self.session.cookies.set(**store_cookie)
```

Reply:
Indeed. My version differs more than I thought, thank you! This is already the third time I've had to change the authentication parameters, it seems some of my past requests went through changes)

Reply:
> вы не ошибаетесь, правильно было бы использовать именно так
> 
> ```
> for name in ['steamLoginSecure', 'sessionid', 'steamRefresh_steam', 'steamCountry']:
>       cookie = self.session.cookies.get_dict()[name]
> 
>       if name == 'steamLoginSecure':
>           community_cookie = create_cookie(name, community_cookie_dic['steamLoginSecure'], community_domain)
>           store_cookie = create_cookie(name, store_cookie_dic['steamLoginSecure'], store_domain)
>       else:
>           community_cookie = create_cookie(name, cookie, community_domain)
>           store_cookie = create_cookie(name, cookie, store_domain)
> 
>       self.session.cookies.set(**community_cookie)
>       self.session.cookies.set(**store_cookie)
> ```
I was wrong, it would be more correct to use it this way
`for name in ('steamLoginSecure', 'sessionid', 'steamRefresh_steam', 'steamCountry'):`
it was like that in the source code
```
def set_sessionid_cookies(self):
        community_domain = SteamUrl.COMMUNITY_URL[8:]
        store_domain = SteamUrl.STORE_URL[8:]
        for name in ('steamLoginSecure', 'sessionid', 'steamRefresh_steam', 'steamCountry'):
            cookie = self.session.cookies.get_dict()[name]
            community_cookie = create_cookie(name, cookie, community_domain)
            store_cookie = create_cookie(name, cookie, store_domain)
            self.session.cookies.set(**community_cookie)
            self.session.cookies.set(**store_cookie)

```